sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -126,24 +126,15 @@ cognitive services.
 
 ## Key concepts
 
-### Client
-The Text Analytics client library provides a [TextAnalyticsClient](https://aka.ms/azsdk-python-textanalytics-textanalyticsclient) to do analysis on batches of documents.
+### Client batched operations
+The Text Analytics client library provides a [TextAnalyticsClient](https://aka.ms/azsdk-python-textanalytics-textanalyticsclient) to do analysis on [batches of documents](#Examples "examples").
```

Comment:
It might be good to add something in here about how this is the recommended way to go when working on a production environment